content/en/tracing/guide/resource_based_sampling.md

```diff
@@ -12,9 +12,7 @@ further_reading:
   text: "Ingestion Control Page"
 ---
 
-{{< callout url="https://www.datadoghq.com/private-beta/resource-based-sampling-adaptive-sampling/" d_target="#signupModal" btn_hidden="true" btn_hidden="false" header="Request access to the beta!" >}}
-Remotely configured sampling rules are in Beta. To request access, complete the form.
-{{< /callout >}}
+<div class="alert alert-info">Remotely configured sampling rules are now GA.</div>
```

Comment:
```suggestion
```
I think we should delete this, because unless otherwise noted, everything in the documentation is GA.